lmnet/executor/evaluate.py

```diff
@@ -150,7 +150,15 @@ def evaluate(config, restore_path, output_dir):
     )
     validation_writer.add_summary(metrics_summary, last_step)
 
+    is_tfds = "tfds_kwargs" in config.DATASET
+    dataset_name = config.DATASET.tfds_kwargs["name"] if is_tfds else config.DATASET_CLASS.__name__
+    dataset_path = config.DATASET.tfds_kwargs["data_dir"] if is_tfds else ""
+
     metrics_dict = {
+        'task_type': config.TASK.value,
+        'model_name': config.NETWORK_CLASS.__name__,
```

Comment:
In Blueoil, we often use `model_name` as config name.
https://github.com/blue-oil/blueoil/blob/master/blueoil/generate_lmnet_config.py#L120

I think It's better to use `network_name`.  How about?

Reply:
@iizukak Thanks! I think it is better too.
Changed 👍